src/main/java/wolox/training/model/dto/BookDTO.java

```diff
@@ -0,0 +1,57 @@
+package wolox.training.model.dto;
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.